### DISCUSSION TITLE
Seeking Guidance on Migrating a WordPress Website to Magento for Better Performance

### DISCUSSION THREAD

**Discussion by ElsieSchreiber** [community/community#146972] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everyone,

I am currently managing a custom packaging website and it's hosted on WordPress. While WordPress has been great for getting started, I've been experiencing limitations in speed, page load times and scalability. Therefore, I’m planning to migrate the website to Magento.

My goal is to optimize performance and introduce a new feature allowing customers to choose from premade packaging options directly on the website.

I’d love to hear from anyone who has experience with:

Migrating a website from WordPress to Magento.
Optimizing Magento for speed and performance (e.g., caching, CDNs).
Configuring product options for premade items in Magento.
Any advice, best practices or recommended tools for making the migration process smooth would be greatly appreciated. If it helps provide context, the website I’m working on is focused on offering [Kraft Paper Studio](https://customkraftpaper.com/).

Looking forward to learning from your experiences!

Thank you in advance.

**Comment by mecodeatlas** [comment]
Hi @ElsieSchreiber,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Magento Support](https://support.magento.com/hc/en-us). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
